forms/i18n/damaging_and_goodfaith/en.json

```diff
@@ -1,16 +1,16 @@
 {
 	"@metadata": [],
 	"damaging-help": "Would you revert this edit?",
-	"damaging-label": "Damaging?",
-	"damaging-no-label": "No",
-	"damaging-no-tooltip": "No, this edit is not damaging and can be kept as is.",
-	"damaging-yes-label": "Yes",
-	"damaging-yes-tooltip": "Yes, this edit is damaging and should be reverted.",
+	"damaging-label": " ",
+	"damaging-no-label": "Not damaging",
+	"damaging-no-tooltip": "This edit can be kept as is",
+	"damaging-yes-label": "Damaging",
+	"damaging-yes-tooltip": "This edit should be reverted",
 	"goodfaith-help": "Does it appear as though the author of this edit was trying to contribute productively?",
-	"goodfaith-label": "Good faith?",
-	"goodfaith-no-label": "No",
-	"goodfaith-no-tooltip": "No, this edit appears to have been made in bad-faith.",
-	"goodfaith-yes-label": "Yes",
-	"goodfaith-yes-tooltip": "Yes, this edit appears to have been made in good-faith.",
+	"goodfaith-label": " ",
+	"goodfaith-no-label": "Bad-faith",
+	"goodfaith-no-tooltip": "Apparently, the author was trying to cause damage intentionally",
+	"goodfaith-yes-label": "Good-faith",
+	"goodfaith-yes-tooltip": "Apparently, the author was trying to contribute productively",
```

Comment:
I'd take out the "Apparently", but otherwise, it looks great.  

If you still want to hedge, maybe you could say, "It seems like this editor was trying to cause damage intentionally." and "It seems like this editor was trying to contribute productively."